src/argilla/client/users.py

```diff
@@ -0,0 +1,310 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import warnings
+from datetime import datetime
+from typing import TYPE_CHECKING, Iterator, List, Optional, Union
+from uuid import UUID
+
+from argilla.client import active_client
+from argilla.client.sdk.commons.errors import (
+    AlreadyExistsApiError,
+    BaseClientError,
+    NotFoundApiError,
+)
+from argilla.client.sdk.users import api as users_api
+from argilla.client.sdk.users.models import UserCreateModel, UserModel
+from argilla.server.models import UserRole
```

Comment:
Same here, this class definition should be in the `sdk.users.models` module

Reply:
I agree, it was something temporary that I forgot to update!